README.md

```diff
@@ -47,6 +47,8 @@ With the settings object you can change the behaviour of the generator. Hopefull
 		vmlBackground: true,
 		rolePresentation: false,
 		lineLength: 799,
+		encodeUrls: true,
+		"lang": "fr"
```

Comment:
We did not use "-s at the prop names in the example. Plz don't mix styles :)

Reply:
If you sure that it works without "s" then I'll change that property everywhere.
example: https://github.com/EDMdesigner/email-generator-docs/blob/master/document-settings/urlEncoding.md

Reply:
my bad, I'll fix these